.github/workflows/publish.yml

```diff
@@ -35,7 +35,7 @@ jobs:
         with:
           node-version: 12
           registry-url: https://registry.npmjs.org/
-      - run: yarn install
+      - run: npm install
```

Comment:
Same as above. It would be better to have a ticket for upgrading the node-versions and have this in a separate PR after we test for publishing.

Reply:
Same comment as above.